AlphaWallet/Settings/ViewControllers/ServersViewController.swift

```diff
@@ -3,11 +3,23 @@
 import UIKit
 
 enum ServerSelection {
+    case unselected
```

Comment:
Why do we need to introduce `.unselected`?

Reply:
nope, missed to remove it, thanks!